src/univers/nuget.py

```diff
@@ -164,6 +163,9 @@ def __lt__(self, other):
         # Revision is the same, so ignore it for comparison purposes.
         return self._base_semver < other._base_semver
 
+    def __hash__(self):
```

Comment:
This may be best in another PR together with a test that this is indeed hashable (e.g. usable as a set value and comparable by id with "is")

Reply:
Okay, will move this to another PR along with the tests.